client/app/main/school-reports/school-reports.html

```diff
@@ -15,7 +15,7 @@
   </div>
   <div class="heading row">
     <h3>Chronically Absent
-      <div class="badge">{{arcaCount * 100 | number: 1}}&#37;</div>
+      <span class="badge">{{arcaCount}}</span>
     </h3>
```

Comment:
Good catch. I was confusing the count with the actual calculation of the percentage.